stories/icons.md

```diff
@@ -1 +1,27 @@
-TODO -- Add some information on how this component should be used.
+## Code examples
+
+### Most basic
+
+```
+<Icon name="cr-logo" />
+```
+
+The most comprhensive list of icons available in arbor is in fort awesome: https://fortawesome.com/kits/27af4258/icons
+
+## Adding new icons
+
+New icons can be added to the fort awesome arbor kit by either uploading an icon, or picking from various icons already available on fort awesome
+https://fortawesome.com/kits/27af4258
```

Comment:
https://github.com/CatchRelease/catchandrelease/wiki/FortAwesome has instructions for this step as well